packages/tinacms/src/components/media/media-manager.tsx

```diff
@@ -474,6 +501,18 @@ const UploadButton = ({ onClick, uploading }: any) => {
   )
 }
 
+const CustomActionButton = ({ name, onClick }: { key: number, name: string, onClick: () => void }) => {
```

Comment:
looks like nothing is being done with `key`

Reply:
Oops, let's use it on the buttons